releng/release-ga-check

```diff
@@ -12,19 +12,24 @@ if __name__ == '__main__':
     parser.add_argument('--no-docker', dest='docker', default=True, action='store_false')
     parser.add_argument('--promote-path', default="")
     parser.add_argument('--ga-version', required=True)
+    parser.add_argument('--chart-version', required=True)
```

Comment:
You also need to edit `builder/builder.mk` `release/ga-check` to add this flag when calling it.

Reply:
D'oh! Nice catch. Done.